static/app/views/issueDetails/metricIssues/metricIssueChart.tsx

```diff
@@ -159,6 +159,7 @@ function MetricIssuePlaceholder({type}: {type: 'loading' | 'error'}) {
 const MetricChartSection = styled('div')`
   display: block;
   padding-right: ${space(1.5)};
+  padding-left: ${space(1.5)};
```

Comment:
nit: These can be updated since we editing this file anyway to avoid `space()` and use `theme.space.lg` (in this case)